internal/action/tab.go

```diff
@@ -124,6 +124,11 @@ func (t *TabList) HandleEvent(event tcell.Event) {
 					return
 				}
 			}
+		case tcell.ButtonNone:
+			if t.List[t.Active()].release {
+				// Mouse release received, while already released...reset mouse
+				t.ResetMouse()
```

Comment:
Yep, we can. the `ButtonNone` event in the `Tab`'s `HandleEvent` does what it needs to do in case the release was for the active `Tab`.
So I will add the `return`.